ray-curator/ray_curator/models/prompt_formatter.py

```diff
@@ -0,0 +1,88 @@
+from typing import Any
+
+import torch
+from transformers import AutoProcessor
+
+VARIANT_MAPPING = {
+    "qwen": "Qwen/Qwen2.5-VL-7B-Instruct",
+    "phi4": "microsoft/Phi-4-multimodal-instruct"
```

Comment:
Do we really support this? It seems like we have hardcoded qwen everywhere right now in our code? Which is ok for now but just trying to understand